src/app/layouts/ModalWindow.js

```diff
@@ -0,0 +1,34 @@
+export class ModalWindow {
+  constructor() {
+    const modalWindow = document.createElement('div');
+    modalWindow.classList.add('modalWindow');
+    modalWindow.id = 'modalWindow';
+
+    const parent = document.getElementById('swquiz-app');
+    parent.appendChild(modalWindow);
+  }
+
+  show(content) {
+    // ""Komponent umożliwia wyświetlenie okna modalnego z zawartością przekazaną jako inny komponent na wejściu do funkcji.""
+    // wiadomo jaki będzie typ komponentu jaki przyjdzie? Bo if (content) to chyba średni warunek...
+    if (content) {
+      content.classList.add('modalWindow__content');
+
+      // create background for content
+      const contentBackground = document.createElement('div');
```

Comment:
Fajnie byłoby to całe tworzenie content backround dac do funkcji. Byłoby bardziej czytelnie :) 
I to w sumie nie jest `content background` bo ten jest... biały. To się mówi chyba overlay na coś takiego.
I potem tylko zastosować funkcję withOverlay na oknie.

```
withOverlay(modalWindow){
  modalWIndow.appendChild(ModalOverlay())
}

function ModalOverlay(){
      const modalOverlay = document.createElement('div');
      modalOverlay.classList.add('modalWindow__overlay');
      modalOverlay.id = 'modalWindow-bg';
     return modalOverlay.
}
```